fix vulnerabilities and webpack version

KavyaReddy732 opened the pull request:
changed the versions of babel-loader,dotenv-webpack,webpack,webpack-cli and the devdependencies 
   "case-sensitive-paths-webpack-plugin": "^2.3.0",
    "concurrently": "^7.0.0",
    "install-peers": "^1.0.3",
    "nodemon": "^2.0.15",
    "sequelize-cli": "^6.4.1",
    "webpack-dev-server": "^4.7.4"

KavyaReddy732 committed:
fix vulnerabilities and webpack version 

changed the versions of babel-loader,dotenv-webpack,webpack,webpack-cli and the devdependencies 
   "case-sensitive-paths-webpack-plugin": "^2.3.0",
    "concurrently": "^7.0.0",
    "install-peers": "^1.0.3",
    "nodemon": "^2.0.15",
    "sequelize-cli": "^6.4.1",
    "webpack-dev-server": "^4.7.4"

orhantoy commented on package.json:
```diff
@@ -48,14 +48,16 @@
+    "webpack-dev-server": "^4.7.4"
   }
 }
+
```
I think we can remove this empty line.

orhantoy reviewed (approved):
Looks good 🎉 
How did you test the changes?

KavyaReddy732 committed:
updated the webpack.config.js

removed the  node: {
    net: 'empty',
    tls: 'empty',
    dns: 'empty',
  }, 
as it's throwing configuration.node should be one of these. changed to this 
node: {
    global: false,
    __filename: false,
    __dirname: false,
  },
and changed the module to 
 use: {
          loader: 'url-loader',
          options: { limit: 100000 }
        }, (Query arguments on 'loader' has been removed in favor of the 'options' property (at ruleSet[1].rules[2].loader: url-loader?limit=100000))
devServer publicpath to static.

KavyaReddy732 committed:
Update package.json

removed extra line

KavyaReddy732 commented:
Tested using the existing DB fo my mealsharing app. i made some changes in webpack.config as the new version of package doesn't allow some of the modules.

KavyaReddy732 commented:
<img width="1440" alt="Screenshot 2022-03-30 at 16 09 12" src="https://user-images.githubusercontent.com/66561373/160854827-60926d42-f3f8-48b9-8a64-895d00cfd630.png">
<img width="1440" alt="Screenshot 2022-03-30 at 16 10 02" src="https://user-images.githubusercontent.com/66561373/160854923-fbaf986d-411d-456e-a2d6-1f97b3c14423.png">

orhantoy commented:
Thanks for fixing this @KavyaReddy732 🎉
